### PR TITLE
Add memory tests

### PR DESCRIPTION
**Pull request by samccone**
it works!
https://travis-ci.org/twosigma/beaker-notebook/builds/73565086#L3970-L4009

### CODE REVIEW

File: test/memory-tests.js [hunk 1]
```diff
@@ -0,0 +1,162 @@
+var drool = require('drool');
```
Comment (scottdraves) [inline test/memory-tests.js:17]: needs banner

Comment (samccone) [inline test/memory-tests.js:17]: good catch! one moment

File: .travis.yml [hunk 2]
```diff
@@ -1,10 +1,15 @@
 language: java
 jdk:
  - oraclejdk7
```
Comment (scottdraves) [inline .travis.yml:3]: should be 8

Comment (scottdraves) [inline .travis.yml:3]: i filed a separate issue for this

**Pull request merged**